### PR TITLE
Update page title (fixes #321)

### LINKED ISSUE #321: Consider updating HTML title to better reflect the page visited
If I have multiple glean dictionary URLs open in my browser they all appear the same. Consider some naming convention that reflects the page the user is on, whether it's a metric detail page or app detail, to help differentiate.

### PR DESCRIPTION
**Pull request by Iinh**
Add HTML head title to each page to fix #321.
 
### Pull Request checklist

- [x] The pull request has a descriptive title (and a reference to an issue it
      fixes, if applicable)
- [x] All tests and linter checks are passing
- [x] The pull request is free of merge conflicts

### CODE REVIEW

File: src/pages/MetricDetail.svelte [hunk 1]
```diff
@@ -6,12 +6,15 @@
...
   import helpText from "../data/help";
   import PageTitle from "../components/PageTitle.svelte";
+  import { pageTitle } from "../state/stores";
 
   export let params;
   let metricName = params.metric.replaceAll("-", ".");
 
   const metricDataPromise = getMetricData(params.app, metricName);
 
+  pageTitle.set(`${metricName} | Metric `);
```
Comment (wlach) [inline src/pages/MetricDetail.svelte:16]: Let's include the app name -- I think *that* something is a metric is usually fairly obvious from the title.

```suggestion
  pageTitle.set(`${metricName} | ${params.app} `);
```

File: src/pages/PingDetail.svelte [hunk 2]
```diff
@@ -9,6 +10,8 @@
 
   export let params;
   const pingDataPromise = getPingData(params.app, params.ping);
+
+  pageTitle.set(`${params.ping} | Ping `);
```
Comment (wlach) [inline src/pages/PingDetail.svelte:14]:
```suggestion
  pageTitle.set(`${params.ping} | ${params.app}`);
```

File: src/pages/TableDetail.svelte [hunk 3]
```diff
@@ -21,6 +22,7 @@
   );
 
   const updateURL = () => dispatch("updateURL", queryString);
+  pageTitle.set(`${params.app}.${params.table} | BigQuery Table `);
```
Comment (wlach) [inline src/pages/TableDetail.svelte:25]:
```suggestion
  pageTitle.set(`${params.table} table | ${params.app}`);
```

File: src/pages/AppList.svelte [hunk 4]
```diff
@@ -19,6 +20,8 @@
     filteredApps = apps.filter((appItem) => appItem.name.includes(filterText));
   }
   let showDeprecated = false;
+
+  pageTitle.set(`Home | Glean Dictionary`);
```
Comment (wlach) [inline src/pages/AppList.svelte:24]:
```suggestion
  pageTitle.set("Glean Dictionary");
```

**Review by wlach — CHANGES_REQUESTED** [review]
I would change some of the titles slightly, but in general this looks great!

File: src/pages/AppDetail.svelte [hunk 5]
```diff
@@ -8,10 +8,13 @@
   import Pill from "../components/Pill.svelte";
   import { TabGroup, Tab, TabContent } from "../components/tabs";
   import PageTitle from "../components/PageTitle.svelte";
+  import { pageTitle } from "../state/stores";
 
   export let params;
 
   const appDataPromise = getAppData(params.app);
+
+  pageTitle.set(`${params.app} | Apps `);
```
Comment (Iinh) [inline src/pages/AppDetail.svelte:17]: @wlach Should I change this to just ` pageTitle.set(params.app);`

Comment (wlach) [reply src/pages/AppDetail.svelte:17]: Yes please! Sorry, missed that one.

**Review by wlach — APPROVED** [review]
Awesome!

**Pull request merged**